regression/cbmc/phi-merge_uninitialized_values/dynamic.desc

```diff
@@ -1,4 +1,4 @@
-CORE  new-smt-backend
```

Comment:
Nit pick (here and elsewhere): for some reason there are two spaces here, and the (automated) changes leave one of them behind, meaning there is a space at the end of the line.